invenio/modules/workflows/views/holdingpen.py

```diff
@@ -257,51 +263,69 @@ def restart_record(objectid, start_point='continue_next'):
         Workflow.uuid == bwobject.id_workflow).first()
 
     start_delayed(workflow.name, [bwobject.get_data()])
-    return 'Record Restarted'
+    return jsonify({
+        "category": "success",
+        "message": unicode(_("Object restarted successfully."))
```

Comment:
Without it, the jsonifier would have problems dealing with the special lazy string type returned from  `_` and cause an error. I guess I could use anyway the `unicodifier`.